English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_numline_dota_20_end.txt

```diff
@@ -0,0 +1 @@
+What number is shown by . A?
```

Comment:
Why do you need the full stop here?

Reply:
These full stops are intentional as per the English audio.